### PR TITLE
chore(readme) Refer to the right release 2.1.0. #20

### PR DESCRIPTION
**Pull request by skounis**

### CODE REVIEW

**Review by ryanbnl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This repository contains the value sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema).
 
-Release: 1.3.0
+Release: 2.1.0
```
Comment (dslmeinte) [inline README.md:5]: `main` is at release tagged `2.2.0`. I thought the v1.4 update was approved?

Comment (skounis) [reply README.md:5]: @dslmeinte The readme is updated and already refers to the latest release `2.2.0`
- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/tree/main
- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/tree/release/2.2.0

This PR is for the `relase/2.1.0` branch where we missed to mark it in the readme
- https://github.com/ehn-dcc-development/ehn-dcc-valuesets/tree/release/2.1.0

image

Comment (dslmeinte) [reply README.md:5]: Uh, that's a clear sign I need another coffee!

**Review by dslmeinte — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dslmeinte) [reply README.md:5]: But more seriously: this commit wasn't signed, which blocks merging.

Comment (skounis) [reply README.md:5]: @dslmeinte I reworked and signed the commit.

**Review by ryanbnl — APPROVED** [review]

**Review by dslmeinte — APPROVED** [review]

**Pull request merged**